priv/scripts/otp/otp.sh

```diff
@@ -5,12 +5,23 @@ set -euox pipefail
 ref_name=$1
 ref=$2
 linux=$3
+arch=$4
```

Comment:
More changes is needed in this file. As it is now the amd64 and arm64 will upload to the same place and clobber each other. The arm64 need to be uploaded to a different directory.

Reply:
Let's use the `builds/otp/#{arch}/#{linux}"` paths as suggested above https://github.com/hexpm/bob/pull/152#discussion_r1600545391.